### PR TITLE
Generate TextAttributes

### PR DESCRIPTION
**Pull request by EPashkin**
Only visible difference is removing `mut_override` in `copy`

### CODE REVIEW

File: src/auto/text_attributes.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    pub struct TextAttributes(Shared<ffi::GtkTextAttributes>);
+
+    match fn {
+        ref => |ptr| ffi::gtk_text_attributes_ref(ptr),
+        unref => |ptr| ffi::gtk_text_attributes_unref(ptr),
+    }
+}
+
+impl TextAttributes {
+
```
Comment (GuillaumeGomez) [inline src/auto/text_attributes.rs]: Since when do we generate an empty line in here? :-/

Comment (EPashkin) [inline src/auto/text_attributes.rs]: Somewhere between 15fe1aa and 34ea1b9 in gir :(

Comment (GuillaumeGomez) [inline src/auto/text_attributes.rs]: I'll let you fix it. ;) Meanwhile, I merge this one.

Comment (EPashkin) [inline src/auto/text_attributes.rs]: https://github.com/gtk-rs/gir/commit/cac0a0f47b424044c3bd066 ;)

**Comment by GuillaumeGomez** [comment]
Thanks!

File: src/auto/text_attributes.rs (hunk 1 above)
Comment (EPashkin) [inline src/auto/text_attributes.rs]: Ok.

**Pull request merged**